app/views/api/v1/merchants/revenue/show.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.extract! @merchant
+if params[:date]
+  json.revenue to_price(@merchant.revenue_on_date(formatted_date))
+else
+  json.revenue to_price(@merchant.revenue)
```

Comment:
@ryanflach I can definitely see the case for serializers in situations like this where you would be able to tell the code where to go (ie which serializer) vs. asking.